bitcoind-rpc-test/src/test/scala/org/bitcoins/rpc/v21/BitcoindV21RpcClientTest.scala

```diff
@@ -0,0 +1,222 @@
+package org.bitcoins.rpc.v21
+
+import org.bitcoins.commons.jsonmodels.bitcoind.RpcOpts.WalletFlag
+import org.bitcoins.commons.jsonmodels.bitcoind._
+import org.bitcoins.core.config.RegTest
+import org.bitcoins.core.gcs.{BlockFilter, FilterType}
+import org.bitcoins.core.psbt.PSBT
+import org.bitcoins.crypto.ECPublicKey
+import org.bitcoins.rpc.client.common.BitcoindVersion
+import org.bitcoins.rpc.client.v21.BitcoindV21RpcClient
+import org.bitcoins.testkit.rpc.BitcoindRpcTestUtil
+import org.bitcoins.testkit.util.BitcoindRpcTest
+
+import java.io.File
+import java.nio.file.Files
+import scala.concurrent.Future
+
+class BitcoindV21RpcClientTest extends BitcoindRpcTest {
+
+  lazy val clientPairF: Future[(BitcoindV21RpcClient, BitcoindV21RpcClient)] =
+    BitcoindRpcTestUtil.createNodePairV21(clientAccum)
+
+  lazy val clientF: Future[BitcoindV21RpcClient] = clientPairF.map(_._1)
+
+  clientF.foreach(c => clientAccum.+=(c))
```

Comment:
don't you want to add the `clientPairF` to the accumulator? Seems you are only adding one client here

Reply:
`createNodePairV21` adds  them to it